src/sensors.cpp

```diff
@@ -1,261 +1,201 @@
 #include "sensors.h"
-#include "driver/pcnt.h"
-#include "driver/adc.h"
-#include "esp_adc_cal.h"
-#include "driver/timer.h"
+#include "esp_adc/adc_cali.h"
+#include "esp_adc/adc_cali_scheme.h"
+#include "esp_adc/adc_oneshot.h"
 #include "freertos/FreeRTOS.h"
 #include "freertos/task.h"
 #include "freertos/semphr.h"
 #include "esp_log.h"
 #include "board_config.h"
 #include "nvs/eeprom_config.h"
 #include "esp_check.h"
+#include "driver/gptimer.h"
+#include "driver/pulse_cnt.h"
 
 #define PULSES_PER_REV 60 // N2 and N3 are 60 pulses per revolution
-#define PCNT_H_LIM 1
+#define MAX_RPM_PCNT 10000
```

Comment:
I would suggest to make this a configurable value.

Reply:
True. I think we should discuss at a later date (In Dev branch), having a settings configuration for variables in the code. Not necessarily a map data structure, but almost something like "Advanced options" for debugging purposes only.